source/includes/authenticated_api/_intro.md

```diff
@@ -2,28 +2,25 @@
 
 The Authenticated REST API allows customers to build applications that interact with their own data or securely grant access to third-party app developers without exposing administrative credentials. It is designed to be consumed server-side, in contrast to the JSONP API which is designed for unauthenticated javascript integrations.
 
-We use [OAuth2](http://oauth.net/2/) to allow organizations to securely grant access to API Applications. OAuth2 is a standardized protocol that allows user credentials to be exchanged for a secret token which can then be used to access an API. There are OAuth2 implementations available for most common programming languages. You'll probably want to read some general documentation on how OAuth2 works before attempting to use this API.
-
-An *API Application* must be configured within your ControlShift instance before using the API. Access is then granted to this application, and exchanged for a token which is used to authenticate your access to API endpoints.
-
 If you're synchronizing data from ControlShift into another system then the Webhooks API is likely a better choice for that use case. 
 
-## Rate Limits
+## Getting Started
+We support both API keys and OAuth2 for authenticating with the API. API keys are the recommended approach, because they are simpler to set up.
```

Comment:
Should we mention that we plan on removing support for OAuth2 auth sometime in the future to push users more strongly to use API keys?

Reply:
Hmm, good question! I'm not sure how definitely to say that, since it's not something we're doing _soon_, but I can add a note that OAuth2 support might be deprecated in the future